### ISSUE TITLE
PHP Fatal error composer

### ISSUE THREAD

**Issue by Sabajanes** [glpi-project/glpi#13989]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.5

### Bug description

With an installation that worked correctly now gives error 500 with the following log

### Relevant log output

```shell
[Thu Feb 02 15:49:29.413379 2023] [php:warn] [pid 1124] [client xxx.xxx.xxx.xxx:52159] PHP Warning:  require(/var/www/html/glpi/vendor/composer/../htmlawed/htmlawed/htmLawed.php): Failed to open stream: No such file or directory in /var/www/html/glpi/vendor/composer/autoload_real.php on line 53
[Thu Feb 02 15:49:29.413445 2023] [php:error] [pid 1124] [client xxx.xxx.xxx.xxx:52159] PHP Fatal error:  Uncaught Error: Failed opening required '/var/www/html/glpi/vendor/composer/../htmlawed/htmlawed/htmLawed.php' (include_path='.:/usr/share/php') in /var/www/html/glpi/vendor/composer/autoload_real.php:53\nStack trace:\n#0 /var/www/html/glpi/vendor/composer/autoload_real.php(36): composerRequire65fed94925510513db9a9a974588ab71()\n#1 /var/www/html/glpi/vendor/autoload.php(25): ComposerAutoloaderInit65fed94925510513db9a9a974588ab71::getLoader()\n#2 /var/www/html/glpi/inc/autoload.function.php(426): require_once('...')\n#3 /var/www/html/glpi/inc/based_config.php(48): include_once('...')\n#4 /var/www/html/glpi/inc/includes.php(42): include_once('...')\n#5 /var/www/html/glpi/marketplace/glpiinventory/index.php(34): include('...')\n#6 {main}\n  thrown in /var/www/html/glpi/vendor/composer/autoload_real.php on line 53
```

**Comment by trasher** [comment]
You've downloaded source file archive, that's not the good one.

**Comment by trasher** [comment]
Also, 10.0.5 is outdated...

**Comment by cedric-anne** [comment]
Hi,

It looks that someone deleted the `/var/www/html/glpi/vendor/htmlawed/htmlawed/htmLawed.php` file on your server. Probably someone that misread the workaround for https://github.com/glpi-project/glpi/security/advisories/GHSA-c5gx-789q-5pcr security advisory (that was already fixed as you used the 10.0.5 version).
You should upgrade to GLPI 10.0.6, it will fix this issue and some other ones.
